site/docs/samples/notebook-write.md

```diff
@@ -0,0 +1,463 @@
+      helm install localstack localstack-charts/localstack --set startServices="s3" --set service.type=ClusterIP
+      kubectl wait --for=condition=ready --all pod -n fybrik-notebook-sample --timeout=120s
+      ```
+      create a port-forward to communicate with localstack server:
+      ```bash
+      kubectl port-forward svc/localstack 4566:4566 &
+      ```
+    3. Use [AWS CLI](https://aws.amazon.com/cli/) to configure localstack server:
+      ```bash
+      export ENDPOINT="http://127.0.0.1:4566"
+      aws configure set aws_access_key_id ${ACCESS_KEY} && aws configure set aws_secret_access_key ${SECRET_KEY}
+
+
+## Deploy resources for write scenarios
+
+Deploy [Datashim](https://github.com/datashim-io/datashim): 
+```yaml
+kubectl apply -f https://raw.githubusercontent.com/fybrik/fybrik/master/third_party/datashim/dlf.yaml
+```
+
+For more deployment options of Datashim based on your environment please refer to the [datashim site](https://github.com/datashim-io/datashim)
+
+Register the credentials required for accessing the object storage. Replace the values for `access_key` and `secret_key` with the values from the object storage service that you used and run:
+
+```yaml
+cat << EOF | kubectl apply -f -
+apiVersion: v1
+kind: Secret
+metadata:
+  name: bucket-creds
+  namespace: fybrik-system
+type: Opaque
+stringData:
+  access_key: "${ACCESS_KEY}"
+  accessKeyID: "${ACCESS_KEY}"
+  secret_key: "${SECRET_KEY}"
+  secretAccessKey: "${SECRET_KEY}"
+EOF
+```
+Then, register two storage accounts. Replace the values for `region` and `endpoint` with values from the object storage services that you used and run:
+
+```yaml
+cat << EOF | kubectl apply -f -
+apiVersion:   app.fybrik.io/v1alpha1
+kind:         FybrikStorageAccount
+metadata:
+  name: theshire-storage-account
+  namespace: fybrik-system
+spec:
+  id: theshire-object-store
+  region: theshire
+  endpoint: "http://localstack.fybrik-notebook-sample.svc.cluster.local:4566"
+  secretRef:  bucket-creds
+EOF
+```
+```yaml
+cat << EOF | kubectl apply -f -
+apiVersion:   app.fybrik.io/v1alpha1
+kind:         FybrikStorageAccount
+metadata:
+  name: neverland-storage-account
+  namespace: fybrik-system
+spec:
+  id: theshire-object-store
+  region: neverland
+  endpoint: "http://localstack.fybrik-notebook-sample.svc.cluster.local:4566"
+  secretRef:  bucket-creds
+EOF
+```
+
+## Scenarios one: write is forbidden due to governance restrictions
+
+### Define data access policies for write
+
+Define an [OpenPolicyAgent](https://www.openpolicyagent.org/) policy to forbid to write sensitive data to regions `neverland` and `theshire`. This policy prevents the writing as the deployed fybrik storage account resources applied are in `neverland` and `theshire`.
+
+Below is the policy (written in [Rego](https://www.openpolicyagent.org/docs/latest/policy-language/#what-is-rego) language):
+
+```rego
+package dataapi.authz
+
+rule[{"policy": description}] {
```

Comment:
I added a link. Thanks

Reply:
You linked to rego, but perhaps it would be better to link to our page describing how to write opa data governance policies.  https://fybrik.io/dev/tasks/using-opa/